Formula/wb32-dfu-updater_cli.rb

```diff
@@ -0,0 +1,21 @@
+class Wb32DfuUpdaterCli < Formula
+  desc "USB programmer"
```

Comment:
Although descriptions should indeed be very short, this feels too vague.

Other than this looks ready to merge.